SecurityManager/Clients.Web/res/Themes/NovaGray/Image/sprite.svg

```diff
@@ -1,4 +1,4 @@
-<svg xmlns="http://www.w3.org/2000/svg">
+<svg height="16" width="16" xmlns="http://www.w3.org/2000/svg">
```

Comment:
I think we should use a CSS file for this. We don't know if there's ever going to be some other change to the image but keeping things consistent makes things easier to maintain.